integration_tests/models/test_charindex.sql

```diff
@@ -0,0 +1,35 @@
+with Input_data as (
+    select
+        'This is a text!' as `Text`
+)
+
+select
+    {# Find a substring (basic scenario) #}
+    {{ pm_utils.charindex('his', '`Text`') }} as `Find_basic_scenario`,
+    2 as `Find_basic_scenario_expected`,
+
+    {# Find a substring with multiple occurences #}
+    {{ pm_utils.charindex('is', '`Text`') }} as `Find_first_of_multiple_occurrences`,
+    3 as `Find_first_of_multiple_occurrences_expected`,
+
+    {# Find a space. SQL Server handles spaces differently #}
```

Comment:
It's true that SQL Server handles spaces differently, but that's applicable for the `len` function, not the `charindex` function. Maybe rephrase this?

Reply:
Updated the comment to cover it better